apps/portals/src/configurations/adknowledgeportal/synapseConfigs/data.ts

```diff
@@ -9,6 +9,7 @@ const data: SynapseConfig = {
     rgbIndex,
     name: 'Data',
     visibleColumnCount: 10,
+    isRowSelectionVisible: true,
```

Comment:
ADKP also uses a File View, they can use the latest enhancement (made for the ELITE portal)